ui/src/chat/ChatMessage/Author.tsx

```diff
@@ -89,6 +89,9 @@ export default function Author({
       <div onClick={handleProfileClick} className="shrink-0">
         <Avatar
           ship={ship}
+          // The conditional here appears to have scoping
+          // which affects the author avatar's size in both "chats themselves"
+          // as well as within ReferenceRow components.
           size={isReply || isRef ? 'xxs' : 'xs'}
```

Comment:
So I've been trying to figure out how to ensure author/op author components in references are completely uniform. Right now Talk message references render their author avatars as specified, while both Notebook Post + Gallery Item references render their author avatars larger:

<img width="432" alt="Screenshot 2023-03-09 at 1 28 27 AM" src="https://user-images.githubusercontent.com/1195363/223939162-a337f677-34d6-49e7-97e4-4ace5d2e28ee.png">

While digging around across the various Reference-related files and the Author component definition, I came across the conditional on line `95` that momentarily seemed to be the correct value to tweak (it made the Note and Gallery references' avatars smaller), it seems as though this value also affected the size of talk message author components:

https://user-images.githubusercontent.com/1195363/223939587-0f3dbbb2-0465-44e7-847a-11756bb4c1eb.mov

My question regarding this line: Which component is responsible for rendering reference author avatars at the intended size of `xxs`?

Reply:
`isRef` and `isReply` are set one component up in @/components/References/ReferenceBar.tsx. Could probably just set size statically there to xxs, make the size prop optional in Author, and set xs as the default.